yoti/yoti.install

```diff
@@ -60,6 +62,4 @@ function yoti_update_8101(&$sandbox) {
 function yoti_uninstall() {
   // Delete Yoti config data.
   \Drupal::configFactory()->getEditable('yoti.settings')->delete();
-  // Delete Yoti user table.
-  YotiUserModel::deleteYotiUserTable();
```

Comment:
I saw you said we're removing this because `createYotiUserTable()` uses `hook_schema()` in the .install. So does that cover the deletion for us?

Reply:
It will do - from [hook_schema()](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Database%21database.api.php/function/hook_schema/8.6.x):
>The tables declared by this hook will be automatically created when the module is installed, and removed when the module is uninstalled. 

I've also checked it will remove the table for sites that already have the module enabled.